WcaOnRails/app/models/user.rb

```diff
@@ -270,7 +262,7 @@ def self.delegate_status_allows_senior_delegate(delegate_status)
   def not_illegally_demoting_oneself
     about_to_lose_access = !software_team? && !board_member?
     if current_user == self && about_to_lose_access
-      if software_team_was
+      if self.was_team_member?('software')
         errors.add(:admin, "You cannot resign from your role as a software admin team member! Find another person to fire you.")
```

Comment:
Oh whoops! It looks like I missed this message in 24dc4e97e1331a289219fa231a0bf42c1844b622. Could you change that from "software admin team" to just "software team"? Same for https://github.com/cubing/worldcubeassociation.org/blob/master/WcaOnRails/spec/models/user_spec.rb#L78.